test/src/test/kotlin/graphql/nadel/tests/EngineTests.kt

```diff
@@ -133,7 +146,7 @@ private suspend fun execute(
         println()
     }
 
-    printSyncLine("Running ${fixture.name}")
+    printSyncLine("\n$sep\n${fixture.name}\n$sep")
 
     try {
         val nadel: Nadel = Nadel.newNadel()
```

Comment:
Self documenting test runner :)

Reply:
Btw have you tried out https://github.com/felipe-gdr/intellij-agg-plugin/